pkgs/sdk/server/src/LdClient.cs

```diff
@@ -371,8 +372,7 @@ public FeatureFlagsState AllFlagsState(Context context, params FlagsStateOption[
 
             var builder = new FeatureFlagsStateBuilder(options);
```

Comment:
These weren't used at all in this function. They are used internal to the `FeatureFlagsStateBuilder`, presumably they didn't get cleaned up when that was introduced.